headlessBot.js

```diff
@@ -350,5 +416,14 @@ function calculateCanvasIndex(x, y) {
 }
 
 function rgbToHex(r, g, b) {
-	return '#' + ((1 << 24) + (r << 16) + (g << 8) + b).toString(16).slice(1).toUpperCase();
+    return (
+        "#" +
+        ((1 << 24) + (r << 16) + (g << 8) + b)
+            .toString(16)
+            .slice(1)
+            .toUpperCase()
+    );
 }
+
+let rgbaOrderToHex = (i, rgbaOrder) =>
+    rgbToHex(rgbaOrder[i * 4], rgbaOrder[i * 4 + 1], rgbaOrder[i * 4 + 2]);
```

Comment:
Nebylo by tohle obojí lepší na 1 řádek?

Reply:
ESLint to dává na druhý řádek automaticky, asi JS syntaxe.